projects/frontend/shared-components/gui/projects/documentation-ui/.browserslistrc

```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
nit: should we link browser support to the VEP

Reply:
Sounds reasonable.

Reply:
https://github.com/vmware/versatile-data-kit/issues/1643